Common/Texture.cs

```diff
@@ -78,15 +78,15 @@ public Texture(string path)
             // Mipmaps are smaller copies of the texture, scaled down. Each mipmap level is half the size of the previous one
             // Generated mipmaps go all the way down to just one pixel.
             // OpenGL will automatically switch between mipmaps when an object gets sufficiently far away.
-            // This prevents distant objects from having their colors become muddy, as well as saving on memory.
+            // This prevents moiré effects (google them), as well as saving on texture bandwidth.
```

Comment:
maybe a link to a picture showing that effect - in other explanations you also linked to stuff, rather than saying "google it"

Reply:
Added nice links.